content/blog/2-baby-steps-into-vr.md

```diff
@@ -0,0 +1,140 @@
+* [`Input.GetButton("Fire1")`](https://docs.unity3d.com/ScriptReference/Input.GetButton.html "Input.GetButton") => Detect when the Cardboard trigger is down (or mouse left click).
+
+Drag and drop the script on the *Player*. Now you can click to move in the direction you are facing! (when the game is running in the editor, you can look around by holding “alt”). You can test it on your cardboard, just do the same build steps that you did for the [demo scene](https://developers.google.com/vr/unity/get-started-android#build_and_deploy_to_an_android_device "Get started Android").
+
+When moving around you may notice that it's sometimes hard to see if you are actually moving or not, specially if you can't see the cube. To fix that I often draw a chessboard pattern on the plane using a [*Material* with a custom *Shader*](https://docs.unity3d.com/Manual/Shaders.html "Materials, Shaders & Textures").
+First create a folder called *Materials* in your *Assets* folder, then right click on the folder and select `Create > Material`, call it *Chessboard* (or whatever you like). Now create a new *Shader* (`Create > Shader > Unlit Shader`) and open it with a double click. Replace the existing code with this one:
+```c
+Shader "Custom/ChessboardShader" {
+    Properties
+    {
+        _Density ("Density", Range(2,50)) = 30
+    }
+    SubShader
+    {
+        Pass
+        {
+            CGPROGRAM
+            #pragma vertex vert
+            #pragma fragment frag
+
+            struct v2f
+            {
+                float2 uv : TEXCOORD0;
+                float4 vertex : SV_POSITION;
+            };
+
+            float _Density;
+
+            v2f vert (float4 pos : POSITION, float2 uv : TEXCOORD0)
+            {
+                v2f o;
+                o.vertex = mul(UNITY_MATRIX_MVP, pos);
+                o.uv = uv * _Density;
+                return o;
+            }
+            
+            fixed4 frag (v2f i) : SV_Target
+            {
+                float2 c = i.uv;
+                c = floor(c) / 2;
+                float checker = frac(c.x + c.y) * 2;
+                return checker;
+            }
+            ENDCG
+        }
+    }
+}
+```
+I will not explain how this code works as shaders are rather difficult to understand, and out of the  scope of this tutorial. But if you are interested (and I found it very interesting) you can read more about what shaders do and how they work [here](https://docs.unity3d.com/Manual/SL-VertexFragmentShaderExamples.html "Vertex and fragment shader examples") (that's where I got this piece of code).
+
+Now drag and drop your shader on your material, then drag your material on the plane.
+
+![article-img-centered](https://raw.githubusercontent.com/Tengio/tengio.com/master/static/img/blog/0008/2_3_scene_chessboard.png "Scene with chessboard")
+
+TODO finish it :grin:
```

Comment:
The second blog post is not finished yet, I need to add content before it goes live.